src/services/conversionRates/getConversionRatesService.js

```diff
@@ -283,10 +285,13 @@ const getConversionRates = async (app, requestedDate, requestedSymbol = 'ETH') =
 
   const tokens = app.get('activeTokenWhitelist');
   let coingeckoId = '';
-
+  let equivalentSymbol = undefined;
   tokens.forEach(token => {
```

Comment:
Make a map (object) from symbol to token and not iterate over symbols every time

Reply:
Doing this for `coingeckoId` field too ?

Reply:
What is the convention for accessing the config?
as I see in the code the config data is available with `app.get("...")`, so what should we do if we need config and we don't have the `app` instance?(should I import the `app`?)

Reply:
I think using find instead of foreach make code more readable, and maybe map doesn't needed anymore